lib/swaggerValidator/ajvSchemaValidator.ts

```diff
@@ -192,6 +211,36 @@ export const ajvErrorToSchemaValidateIssue = (
   return result;
 };
 
+const shouldRevalidateError = (
```

Comment:
How about have this function called as ReValidateIfNeed? Then put all the logic here returning modified errors array.